pkg/eks/api.go

```diff
@@ -158,6 +159,10 @@ func LoadConfigFromFile(configFile string, cfg *api.ClusterConfig) error {
 		return errors.Wrapf(err, "reading config file %q", configFile)
 	}
 
+	if err := yaml.UnmarshalStrict(data, &api.ClusterConfig{}); err != nil {
```

Comment:
Where is the old unmarshal call? was it the `runtime.Decode()` below? should we keep both?

Reply:
Yes, and we still want to use that one. The output of this one is not stored. A bit of a hack, but couldn't find a way to make `runtime.Decode()` do the same, but it does other things that are useful for us.

Reply:
Oh, but then this will be confusing, can we add a comment that the actual thing is used is `obj` from the `Decode()`?

Reply:
Yes, good idea, I added a comment with more notes.